src/util/ApplicationFlags.js

```diff
@@ -23,21 +23,16 @@ class ApplicationFlags extends BitField {}
 
 /**
  * Numeric application flags. All available properties:
- * * `MANAGED_EMOJI`
- * * `GROUP_DM_CREATE`
- * * `RPC_HAS_CONNECTED`
  * * `GATEWAY_PRESENCE`
  * * `GATEWAY_PRESENCE_LIMITED`
  * * `GATEWAY_GUILD_MEMBERS`
  * * `GATEWAY_GUILD_MEMBERS_LIMITED`
  * * `VERIFICATION_PENDING_GUILD_LIMIT`
  * * `EMBEDDED`
  * @type {Object}
+ * @see {@link https://discord.com/developers/docs/resources/application#application-object-application-flags}
  */
 ApplicationFlags.FLAGS = {
-  MANAGED_EMOJI: 1 << 2,
-  GROUP_DM_CREATE: 1 << 4,
-  RPC_HAS_CONNECTED: 1 << 11,
```

Comment:
https://github.com/discord/discord-api-docs/issues/2435
https://github.com/discordjs/discord.js/pull/5147#issuecomment-781672475

Reply:
Are you sure those are still valid? Seems like the discussion on the issue said that there was no enum but it does exist now and does not include those 3 flags cc @advaith1

Reply:
Then I guess they’re just leftovers? Either way it’s not documented so I don’t think it should be there but I’ll let the @discordjs/the-big-3 decide

Reply:
I'm fine with either, I'm not sure if there is any use case for them in djs.

Reply:
Sho sho, away!